src/client/components/layouts/InstantSearch/InstantSearchLayout.tsx

```diff
@@ -77,6 +77,7 @@ export function InstantSearchLayout(props: InstantSearchLayoutProps) {
     const { status,  results } = useInstantSearch();
     const isSettingUp = results.__isArtificial;
     const [isInstantSearchLoading, setIsInstantSearchLoading] = useState<boolean>(true);
+    const LISTE_DES_RESULTATS_ID = 'listeDesResultatsId';
```

Comment:
```suggestion
    const LISTE_DES_RESULTATS_ID = 'idDeLaListeDesResultats';
```

Peut-être plus clair ?